sparkmagic/sparkmagic/tests/test_remotesparkmagics.py

```diff
@@ -548,3 +548,290 @@ def test_logs_exception():
     assert result is None
     ipython_display.send_error.assert_called_once_with(EXPECTED_ERROR_MSG
                                                        .format(get_logs_method.side_effect))
+
+
+# New Tests for the spark_collect line magic
+
+#@with_setup(_setup, _teardown)
+#def test_spark_collect_bad_command_writes_error():
+#    line = "bad_command"
+#    usage = "Please look at usage of %spark_collect by executing `%spark_collect?`."
+#
+#    magic.spark_collect(line)
+#
+#    ipython_display.send_error.assert_called_once_with("Subcommand '{}' not found. {}".format(line, usage))
+
+
+@with_setup(_setup, _teardown)
+def test_spark_collect_run_cell_command_parses():
+    run_cell_method = MagicMock()
+    result_value = ""
+    run_cell_method.return_value = (True, result_value)
+    spark_controller.run_command = run_cell_method
+
+    name = None
+    line = ""
```

Comment:
This variable is never used in tests. Please remove in all tests where it's not needed.